src/pk-main.c

```diff
@@ -173,10 +173,14 @@ main (int argc, char *argv[])
 	/* get values from the config file */
 	conf = g_key_file_new ();
 	conf_filename = pk_util_get_config_filename ();
+	if (conf_filename == NULL) {
+		g_error ("Cannot found config file.");
```

Comment:
I don't know if we need this g_error() here since we have the g_warning() inside the pk_util_get_config_filename() function, but the text is not appearing to me by default.